packages/block-library/src/template-part/edit/index.js

```diff
@@ -97,21 +84,17 @@ export default function TemplatePartEdit( {
 			// FIXME: @wordpress/block-library should not depend on @wordpress/editor.
 			// Blocks can be loaded into a *non-post* block editor.
 			// eslint-disable-next-line @wordpress/data-no-store-string-literals
-			const defaultWrapperElement = select( 'core/editor' )
+			const areaObject = select( 'core/editor' )
 				.__experimentalGetDefaultTemplatePartAreas()
 				.find( ( { area: value } ) => value === _area )?.area_tag;
 
 			return {
 				innerBlocks: getBlocks( clientId ),
 				isResolved: hasResolvedEntity,
 				isMissing: hasResolvedEntity && isEmpty( entityRecord ),
-				defaultWrapper: defaultWrapperElement || 'div',
+				defaultWrapper: areaObject?.area_tag ?? 'div',
+				areaLabel: areaObject?.label || __( 'Template Part' ),
```

Comment:
Related to the above comment probably we should remove .area_tag from the areaObject definition.